File: C/1020.c
#include <stdio.h>

int main()
{
    int days;
    scanf("%d", &days);
    printf("%d ano(s)\n", days / 365);
    printf("%d mes(es)\n", (days % 365) / 30);
    printf("%d dia(s)\n", (days % 365) % 30);

    return 0;
}

File: PHP/1020.php
<?php
$days = rtrim(fgets(STDIN));
$years = floor($days / 365);
$days = fmod($days, 365);
$months = floor($days / 30);
$days = fmod($days, 30);
echo $years . " ano(s)\n";
echo $months . " mes(es)\n";
echo $days . " dia(s)\n";
